classes/operations/label_manager.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../structure/label'
+require_relative '../../module/data_operations'
+
+class LabelManager
+  include DataOperations
+
+  attr_accessor :label_data
+
+  def initialize
+    @label_data = fetch_data('label')
+  end
+
+  def create_label
+    state = false
+
+    until state
+
+      puts 'Please enter the label title:'
+      title = gets.chomp.to_s
+      state = true unless title == ''
+      puts 'Label title Required' unless state == true
+      title
+    end
+
+    puts 'Enter the color of the label'
+    color = gets.chomp
+
+    new_label = Label.new(title, color)
+
+    @label_data << new_label.to_hash
+
+    update_data('label', new_label.to_hash)
+    new_label
+  end
+
+  def list_labels
+    @label_data.each_with_index do |label, index|
+      puts '---------------------'
+      puts "(#{index}) #{label['id']} - #{label['title']} - #{label['color']})"
+      puts '---------------------'
+    end
+
+    puts "(#{@label_data.length}) - Create a new label"
+  end
+
+  def format_label(hashed_label)
+    label = Label.new(hashed_label['title'])
```

Comment:
Hey @joskalenda, good job so far! you are so close. Thus line of code throws an error when you try to add a book to a label by selecting the corresponding label.

Consider adding `hashed_label['color']` as a second parameter to the instance creation, as the label class expects it